Dockerfile

```diff
@@ -6,10 +8,17 @@ COPY . .
 # just use that.  For now we work around this by copying a tarball.
 RUN make install DESTDIR=./instroot && tar -C instroot -cf instroot.tar .
 
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.14 AS rhel9-builder
```

Comment:
cc OKD folks - is this a problem?  Do we need distinct dockerfiles?

Reply:
This should be fine for Prow builds (i.e. the official OKD builds). It might cause problems when using https://github.com/okd-project/okd-payload-pipeline/ to builds outside of Prow, but we should be able to work around it there, too.

cc @openshift/okd-maintainers @aleskandro